### ISSUE TITLE
[JS Curriculum] Shopping Cart Step 43 missing commutative addition property 

### ISSUE THREAD

**Issue by juancaorg** [freeCodeCamp/freeCodeCamp#54598]
### Describe the Issue

When working through the [Shopping Cart Step 43](https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/learn-basic-oop-by-building-a-shopping-cart/step-43), it's expected from the user to create a `calculateTotal()` method that internally uses a `reduce()` to make a sum, like the following:

```js
calculateTotal() {
  const subTotal = this.items.reduce((total, item) => total + item.price, 0);
}
```

But when presented the following solution, making the reduce's callback sum the other way around, it doesn't pass:

```js
calculateTotal() {
  const subTotal = this.items.reduce((total, item) => item.price + total, 0);
}
```

### Affected Page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/learn-basic-oop-by-building-a-shopping-cart/step-43

### Your code

```js
calculateTotal() {
  const subTotal = this.items.reduce((total, item) => item.price + total, 0);
}
```

### Expected behavior

Because of the [Commutative property of addition](https://www.khanacademy.org/math/cc-sixth-grade-math/cc-6th-expressions-and-variables/properties-of-numbers/a/properties-of-addition), the previous presented solution should work.

### System

- Device: Laptop
- OS: macOS 13.6.6
- Browser: Chrome
- Version: 124

**Comment by jdwilkin4** [comment]
I agree that both versions should be accepted since they are valid solutions.
